modules/appeals_api/app/controllers/appeals_api/v0/appeals_controller.rb

```diff
@@ -66,7 +66,7 @@ def header(key)
 
       def va_profile
         OpenStruct.new(
-          birth_date: header('X-VA-Birth-Date')
+          birth_date: (header('X-VA-Birth-Date') || nil)
```

Comment:
Will EVSS work with a nil birth date? If so why are we requiring it in claims lol

Reply:
we don't need to fall back to nil I agree here, is there a reason this was done? were you getting a blank string value by mistake?

Reply:
I removed the whole va_profile as it is not needed from the appeals side